DataManagementSystem/DB/FileCatalogComponents/FileManagerBase.py

```diff
@@ -1257,14 +1207,14 @@ def _checkLFNPFNConvention( self, lfn, pfn, se ):
     result = self._getStorageElement( se )
```

Comment:
Agreed. It can be just dropped for the moment ( _checkLFNPFNConvention is not actually used ). The way how the SE information is used in the DFC is one more thing to be sorted out. As we discussed, this should be rather moved to the client.